FlowCrypt/Common/Constants.swift

```diff
@@ -18,6 +18,12 @@ enum Constants {
         }
     }
 
+    static let inboxDateFormatter: DateFormatter = {
```

Comment:
I think this one is unused & should be removed (unless I'm wrong). Now we use `convertDate` below.